addons/l10n_hr/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_l10n_hr_kpd_category,access_l10n_hr_kpd_category,model_l10n_hr_kpd_category,account.group_account_manager,1,1,1,1
```

Comment:
I think we can also give read-only access to all users (regardless of what their Accounting rights are).
This data isn't sensitive and could be used across sales, purchases, PoS etc.